etc/helm/pachyderm/values.schema.json

```diff
@@ -566,9 +563,6 @@
                             "type": "string"
                         }
                     }
-                },
-                "usesRoot": {
```

Comment:
Oh, what was this?

Reply:
that was killed in another PR, but looks like the schema for it got missed, I can break that out into another cleanup PR

Reply:
Nah, doing it here is fine.  api port won't be coming back ;)